extensions/eda/plugins/event_source/aws_cloudtrail.py

```diff
@@ -36,8 +36,8 @@
 from datetime import datetime
 from typing import Any
 
-from aiobotocore.client import BaseClient
 from aiobotocore.session import get_session
+from botocore.client import BaseClient
```

Comment:
This seems wrong to me. Changing `botocore` -> `aiobotocore` will change functionality.

Reply:
I doubt it, see https://github.com/aio-libs/aiobotocore/blob/master/aiobotocore/client.py#L3 

Because they do not exported explicitly via __all_, that import is considered not work, so importing it from original location is the right approach.